README.md

```diff
@@ -119,10 +111,50 @@ request = ArticlesClient.create(title: "Hello, World")
 status, headers, body = *request.submit
 ```
 
+### Response body parsing
+
+When `ActionClient` is able to infer the request's `Content-Type` to be either
+`JSON` or `XML`, it will parse the returned `body` value ahead of time.
+
+Responses with `Content-Type: applicaiton/json` headers will be parsed into
+[`Hash` instances][ruby-hash] by [`JSON.parse`][json-parse]. JSON objects will
+be parsed into instances of
+[`HashWithIndifferentAccess`][HashWithIndifferentAccess], so that keys can be
+accessed via `Symbol` or  `String`.
```

Comment:
Typo, but also mentioning both `Hash` and `HashWithIndifferentAccess` seemed contradictory:

```suggestion
Responses with `Content-Type: application/json` headers will be parsed into
Ruby objects by [`JSON.parse`][json-parse]. JSON objects will
become instances of
[`HashWithIndifferentAccess`][HashWithIndifferentAccess], so that keys can be
accessed via `Symbol` or  `String`.
```